src/request.rs

```diff
@@ -462,6 +468,27 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_match_name() {
+        let mut rdef = ResourceDef::new("/index.html");
+        *rdef.name_mut() = "index".to_string();
+
+        let mut rmap = ResourceMap::new(ResourceDef::new(""));
+        rmap.add(&mut rdef, None);
+
+        assert!(rmap.has_resource("/index.html"));
+
+        let req = TestRequest::with_uri("/test")
+            .header(header::HOST, "www.rust-lang.org")
+            .rmap(rmap)
+            .to_http_request();
+        let route_name = req.0.rmap.match_name("index");
```

Comment:
I think this should be `match_name("/index.html")` ?
_Since the method is path in, Option(name) out?_